Src/BootCamp.Chapter/CleanTheFile.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class CleanTheFile
+    {
+        public void Clean(string file, string outputFile)
+        {
+            if (Testers.IsThisStringValid(file) || Testers.IsThisStringValid(outputFile))
+            {
+                throw new ArgumentException();
+            }
+
+            StringBuilder cleanFile = new StringBuilder();
+
+            using (var fileS = new FileStream(file, FileMode.Open))
+            {
+                using (var reader = new StreamReader(fileS))
+                {
+                    foreach (char letter in reader.ReadToEnd())
+                    {
+                        if (letter == '_')
+                        {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove or fill this block of code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)